package.json

```diff
@@ -16,6 +15,7 @@
 		"aud": "^1.1.3",
 		"auto-changelog": "^2.2.1",
 		"eslint": "^7.17.0",
+		"has-symbols": "^1.0.1",
```

Comment:
@ljharb It just occurred to me that this change is wrong, because `has‑symbols` is used in `implementation.js`: <https://github.com/ljharb/util.promisify/blob/8980c559962db489a7720f88bfaa099503ab920f/implementation.js#L25>

Reply:
Indeed; it’s been fixed in #25.